### PR TITLE
apps node group in eks

### PR DESCRIPTION
**Pull request by mmpetarpeshev**

### CODE REVIEW

File: eks-cluster.tf [hunk 1]
```diff
@@ -40,6 +40,27 @@ module "eks" {
...
+    disk_size = 100
+  }
+
+  node_groups = {
+    apps = {
+      desired_capacity = 1
+      max_capacity     = 10
+      min_capacity     = 1
+
+      instance_types = ["m5.large"]
```
Comment (dincho) [inline eks-cluster.tf]: should be a variable

File: eks-cluster.tf [hunk 2]
```diff
@@ -40,6 +40,27 @@ module "eks" {
...
 
+  node_groups_defaults = {
+    ami_type  = "AL2_x86_64"
+    disk_size = 100
+  }
+
+  node_groups = {
+    apps = {
+      desired_capacity = 1
+      max_capacity     = 10
```
Comment (dincho) [inline eks-cluster.tf]: variables please

File: eks-cluster.tf [hunk 3]
```diff
@@ -40,6 +40,27 @@ module "eks" {
...
+      min_capacity     = 1
+
+      instance_types = ["m5.large"]
+      capacity_type  = "ON_DEMAND"
+      k8s_labels     = local.standard_tags
+
+      update_config = {
+        max_unavailable_percentage = 50 # or set `max_unavailable`
```
Comment (dincho) [inline eks-cluster.tf]: also variable

File: eks-cluster.tf (hunk 1 above)
Comment (mmpetarpeshev) [reply eks-cluster.tf]: done

File: eks-cluster.tf (hunk 2 above)
Comment (mmpetarpeshev) [reply eks-cluster.tf]: done

File: eks-cluster.tf (hunk 3 above)
Comment (mmpetarpeshev) [reply eks-cluster.tf]: done

**Pull request merged**